api/app/tests/hfi/test_hfi_endpoint_request.py

```diff
@@ -11,7 +11,7 @@
 import app.routers.hfi_calc
```

Comment:
Dumb question - and maybe not part of this PR - but are we testing anywhere that an expired token kicks you out on the back-end?

Reply:
We have these tests on the backend: https://github.com/bcgov/wps/blob/main/api/app/tests/test_auth.feature